src/index.js

```diff
@@ -0,0 +1,74 @@
+import runMocha from './run-mocha.js';
+import BrowserHelper from './browser-helper';
+import Site from './site';
+
+
+export default class FunctionalHarness {
+    constructor () {
+        this.sources = null;
+
+        this.site = null;
+
+        this.browsersInfo = null;
+
+        this.username  = null;
+        this.accessKey = null;
+        this.jobName   = null;
+
+        this.testTimeout = null;
+    }
+
+    src (sources) {
+        this.sources = sources;
+
+        return this;
+    }
+
+    siteSettings (sitePort1, sitePort2, viewsPath) {

```

Comment:
I've corrected this already: https://github.com/AlexanderMoskovkin/functional-harness/pull/1#discussion_r47750078

`siteSettings - sitePort1, sitePort2...` is redundant. `siteSettings - port1, port2` is ok.